src/mc.rs

```diff
@@ -7,7 +7,7 @@
 // Media Patent License 1.0 was not distributed with this source code in the
 // PATENTS file, you can obtain it at www.aomedia.org/license/patent.
 
-#[cfg(all(target_arch = "x86_64", not(windows), feature = "nasm"))]
+#[cfg(all(target_arch = "x86_64", feature = "nasm"))]
 pub use self::nasm::*;
 #[cfg(any(not(target_arch = "x86_64"), windows, not(feature = "nasm")))]
```

Comment:
Need to update the counter-condition also.
```suggestion
#[cfg(any(not(target_arch = "x86_64"), not(feature = "nasm")))]
```